### DISCUSSION TITLE
Bug on github?

### DISCUSSION THREAD

**Discussion by githubUser01946** [SupaGruen/SD-CheatSheet#8] [General]
I visited this page of yours a few days ago and saw loads of examples of peoples work, now all I see is a list of artists and descriptions.

Is this a bug?

![image](https://github.com/SupaGruen/StableDiffusion-CheatSheet/assets/117453319/0369a1f2-78f7-47a3-b259-c0fb62c86046)

**Comment by SupaGruen** [comment]
This is part of the 'Notes' section, the styles are still there and working.

**Reply by macronomicus** [reply]
The styles page no longer loads, it just happened too, I was using the site like an hour ago, did a page refresh & it now defaults to the Notes page, and all the main menu links only show the Notes page

**Reply by SupaGruen** [reply]
should work again!

**Comment by githubUser01946** [comment]
I've gone and downloaded it, will use that feom now on I think.

**Reply by SupaGruen** [reply]
should work again!

**Reply by githubUser01946** [reply]
> should work again!

Sorry for the late reply. Yes it does, thank you.
